app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, copy_from, copy_to = command.split(" ")
+
+    if cmd != "mv":
+        raise ValueError("Only 'mv' is possible")
+
+    if "/" not in copy_to:
+        copy_to_path = copy_to
```

Comment:
The variable `copy_to_path` is assigned here but not used in the else block. Consider using `os.path.join` to concatenate parts of the path as per the checklist: 'Use os.path.join to concatenate parts of the path. In that case, your app will be cross-platform and will use either / or \ depending on OS'.